src/main/java/com/visenze/datatracking/Constants.java

```diff
@@ -0,0 +1,17 @@
+package com.visenze.datatracking;
+
+public class Constants {
+    public static final String BASE_URL = "https://staging-analytics.data.visenze.com/v3/";
```

Comment:
should update to production URL. It is available: `analytics.data.visenze.com`. Need to support both China and default global endpoints. Note that for China it is `analytics.visenze.com.cn`. Does not have the `.data`.

Reply:
currently country is not a mandtory field in the event Data. Do you mean set country as a mandatory parameter in the init process ?

Reply:
no need, just need a way to provide a custom endpoint and tell them to use the correct method. Alternately, we can have two constructors, 1 for global, 1 for China. No need country.

Reply:
ok add getCNInstance() in dataTracking class